gse-app/src/main/java/com/powsybl/gse/app/NewProjectPane.java

```diff
@@ -63,7 +63,16 @@ public NewProjectPane(Window window, AppData appData, GseContext context) {
         });
         folderTextField.textProperty().bind(Bindings.createObjectBinding(() -> {
             Folder folder = folderProperty.get();
-            return folder != null ? folder.getPath().toString() : null;
+            if (folder == null) {
+                return null;
+            } else {
+                String path = folder.getPath().toString();
+                if (path.substring(path.length() - 1).equals(":")) {
+                    return path + "/";
+                } else {
+                    return path;
+                }
```

Comment:
Instead of checking the name of the folder, could you check if the folder is the root folder (no parent)